inc/cappuccino/utlru_cache.hpp

```diff
@@ -148,6 +148,26 @@ class utlru_cache
         return deleted_elements;
     }
 
+    /**
+     * Removes all elements from the cache (which are destroyed), leaving the container size 0.
+     * @return none
+     */
+    auto clear() -> void
+    {
+        if (m_used_size > 0)
```

Comment:
m_used_size isn't atomic so its probably best to take the lock and then return immediately if its == 0

Reply:
Done.